lib/fuzzy-finder-view.js

```diff
@@ -44,7 +44,7 @@ export default class FuzzyFinderView {
           this.previousQueryWasLineJump = true
           this.selectListView.update({
             items: [],
-            emptyMessage: 'Jump to line in active editor'
+            emptyMessage: 'Enter a <row> or <row>:<column> to go there. Examples: "3" for row 3 or "2:7" for row 2 and column 7'
```

Comment:
Hmm...I'm not that sure.  I'll try to get back to you after discussing whether or not we should deprecate go-to-line.